pkg/logs/status/status.go

```diff
@@ -77,7 +81,17 @@ func RemoveGlobalWarning(key string) {
 	}
 }
 
+// Appends an error message for the status display (errors will stop the agent)
```

Comment:
In go your comment must start with the name of the method as follow:
`// AddGlobalError  appends ...`

Reply:
Ok good to know, I'll update the file.